src/Orchard.Cms.Web/Startup.cs

```diff
@@ -48,7 +48,9 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 loggerFactory.AddDebug();
             }
 
-            app.UseModules();
+            app.UseMiddleware<ModularTenantContainerMiddleware>();
+            app.UseMiddleware<StoreTenantNameAtHttpContextMiddleware>();
```

Comment:
I need the tenant name stored at httpcontext just after the tenant pipeline is initialized to log properly the tenant name on the first request after starting the server. That's why I need to replace `app.UseModules()` per what it does. 
I'm not happy with the solution of not using `app.UseModules()` .
@sebastienros previously I set it within ModularTenantContainerMiddleware but you told me I should avoid to have logging code specific for a log library within OrchardCore. However thinking it twice to store tenant name in httpcontext for sharing it with other middlewares is not specific code for an specific log library. Don't you prefer the previous solution than this one?